tests/integration/fixtures/common.py

```diff
@@ -38,7 +38,21 @@ def is_port_available(port: int, host: str = "localhost") -> bool:
 
 def start_llama_stack_server(config_name: str) -> subprocess.Popen:
     """Start a llama stack server with the given config."""
-    cmd = ["llama", "stack", "run", config_name]
+    cmd = [
```

Comment:
nit: more readable to have a string command and then use `shlex.split()` when passing to `Popen`